test/test.js

```diff
@@ -0,0 +1,298 @@
+const container = require("../index.js").container;
+const assert = require("assert");
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
```

Comment:
What are these last three dependencies for?

Reply:
Dead code! They were only used for the `.load` feature, good catch 💃